models/src/main/scala/coop/rchain/casper/protocol/CasperMessage.scala

```diff
@@ -141,21 +143,26 @@ object BlockMessage {
       bm.postStateHash,
       bm.justifications,
       bm.bonds.map(b => (b.validator, b.stake)).toMap,
-      bm.rejectedDeploys,
+      bm.rejectedDeploys.toSet,
+      bm.rejectedBlocks.toSet,
+      bm.rejectedSenders.toSet,
       state,
       bm.sigAlgorithm,
       bm.sig
     )
 
   def toProto(bm: BlockMessage): BlockMessageProto = {
+    // TODO: Sorting should not be required. Block hash is checked as serialized binary of the block.
```

Comment:
I assume that whoever wrote initial code was thinking about ordering because of deterministic hashing of the block. We don't care about the order, but order can change calculated hash.
But this is not important because block is transmitted as binary blob so it will be checked as is with any order was initially packaged.